docs/website/blog/3.0.0-GA-blog.md

```diff
@@ -0,0 +1,333 @@
+---
+title: odo v3.0.0 GA Released 🎉
+author: Parthvi Vala
+author_url: https://github.com/valaparthvi
+author_image_url: https://github.com/valaparthvi.png
+tags: ["release"]
+slug: odo-v3-release
+---
+
+odo v3 is now GA! 🎉
+
+![](../static/img/logo_small.png)
+<!--truncate-->
+Are you an application developer wanting to work closely with a production like environment? But can't, because you either don't have the time or energy to dive in the steep learning curve involved in running your application on the cluster?
+
+`odo` is here to answer your problems!
+
+
+Over the past couple of months, odo team has been working on creating a better experience with odo.
+
+## So, what's new?
+
+`odo` v3 comes with a new set of commands that replaces the old ones but essentially does the same thing, in a faster and more efficient way.
+
+### odo init
+`odo init` allows you to fetch an appropriate [Devfile](https://devfile.io/docs/2.1.0/what-is-a-devfile) for your project.
+
+```shell
+  __
+ /  \__     Initializing a new component
+ \__/  \    Files: Source code detected, a Devfile will be determined based upon source code autodetection
+ /  \__/    odo version: v3.0.0
+ \__/
+
+Interactive mode enabled, please answer the following questions:
+Based on the files in the current directory odo detected
+Language: javascript
+Project type: nodejs
+The devfile "nodejs" from the registry "StagingRegistry" will be downloaded.
+? Is this correct? Yes
+ ✓  Downloading devfile "nodejs" from registry "StagingRegistry" [3s]
+
+↪ Container Configuration "runtime":
+  OPEN PORTS:
+    - 3000
+  ENVIRONMENT VARIABLES:
+
+? Select container for which you want to change configuration? NONE - configuration is correct
+? Enter component name: my-node-app
+
+Your new component 'my-node-app' is ready in the current directory.
+To start editing your component, use 'odo dev' and open this folder in your favorite IDE.
+Changes will be directly reflected on the cluster.
+```
+
+_How is it different from `odo create` command of v2?_
+* [Detect appropriate Devfile](#detect-appropriate-devfile)
+* [Personalize Devfile configuration](#personalize-devfile-configuration)
+
+#### Detect appropriate Devfile
+`odo init` will automatically analyze your directory and based on the project type, suggest an appropriate Devfile for use.
+
+#### Personalize Devfile configuration
+`odo init` also provides a better interactive way to configure your Devfile according to your project requirement. With this feature, you can modify the environment variables exported to and ports exposed by your application on the cluster.
```

Comment:
It's not really an advanced use case in my opinion since we are providing a way to do this via the command. And as far as I understand, we're trying to balance when we talk about Devfile, in this context, I think it's alright to mention that you can configure your devfile for your project requirements to export envvar and expose ports. I do not see the harm in mentioning devfile.

Reply:
Even the output of `odo init` mentions devfiles. And we can't hide it users will see it as a new file in their repo. It is ok to mention that Devfile exists and odo uses it.